jobs/move_extract.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
```

Comment:
Yes, we need to use it to migrate `--extract=not-local` which includes every `ci-` job